README.md

```diff
@@ -99,6 +100,9 @@ A config file can manage auditing preferences `audit-ci`. The config file's keys
 
 Review the examples section for an [example of config file usage](#example-config-file-and-usage).
 
+> Refrain from using `"directory"` within the config file because `directory`
+> is relative to where the command is run, rather than the directory where the config file exists.
```

Comment:
Do we want to expose the directory option to users of the CLI? I added it purely for the purpose of making unit-testing feasible.

However, as this is the only comment I have (and it's rather tiny) I will think I'll just merge this PR (and we'll just continue the discussion in parallel)